catalogue/webapp/components/ItemRequestModal/ItemRequestModal.tsx

```diff
@@ -454,16 +463,27 @@ const ItemRequestModal: FC<Props> = ({
       }
     } catch (error) {
       setRequestingState('error');
+      setRequestingError(error.description);
       // TODO: error to Sentry?
     }
   }
 
-  function renderModalContent(requestingState: RequestingState) {
+  function renderModalContent(
+    requestingState: RequestingState,
+    requestingErrorMessage?:
+      | string
+      | 'There was a problem requesting this item. Please try again.'
```

Comment:
Why are we duplicating the default error message here? Do we need it in this function signature?

Reply:
Ah yep, that was me messing about making typescript happy as I was trying things out - I'll adjust now!